### PR TITLE
Cust 2140 sidenav theme

### PR DESCRIPTION
**Pull request by jsmylnycky**
CUST-2140 - Revamp sidenav styling
CUST-2127 - Material 0.11.4 fixes (Tab colors, dialog padding, pageSearch, mdDatePicker)
CUST-2009 - Fix font weight of user menu items

### CODE REVIEW

**Comment by rynomars** [comment]
:thumbsup:

**Comment by rwrzesien** [comment]
:+1:

**Comment by darushka** [comment]
:+1:

File: bower.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
     "url": "https://github.com/peachworks/peach.css"
   },
   "dependencies": {
-    "angular-material": "0.11.2",
+    "angular-material": "0.11.3",
```
Comment (darushka) [inline bower.json]: Why 0.11.3 ?

Comment (jsmylnycky) [inline bower.json]: Mistype.  Meant 0.11.4, good catch :)

**Comment by jsmylnycky** [comment]
Released v0.3.0

**Pull request closed without merging**